src/canon/docs_test.jl

```diff
@@ -0,0 +1,26 @@
+
+db = [("Test One", "https://www.google.com", "testone", quote x="AAAA" end)
+     ,("Oscillator", "https://www.archlinux.org", "oscillator", quote y="BBBB" end)]
+
+macro create_pode(modelname,source,variable,pode) 
+  quote 
```

Comment:
For auto docs, you will need to add `export @modelname` to the expr that you are generating.